src/main/webapp/app/request/request-contents.html

```diff
@@ -274,7 +274,7 @@ <h4 class="h4">
             <div ng-if="request.informedConsentFormAttachments.length == 0"><p><i>{{'No forms uploaded.' | translate}}</i></p></div>
             <div class="list-group files" id="icforms">
                 <a ng-repeat="file in request.informedConsentFormAttachments"
-                   ng-href="/api/requests/{{request.processInstanceId}}/informedConsentFormFiles/{{file.id}}"
+                   ng-href="/api/requests/{{request.processInstanceId}}/files/{{file.id}}"
```

Comment:
Shouldn't it be updated also for the edit request [(edit-request.html#L508)](https://github.com/thehyve/dntp-portal/blob/dev/src/main/webapp/app/request/edit-request.html#L508)?

Reply:
A, yes, nice catch!